src/main/java/io/github/opencubicchunks/cubicchunks/core/worldgen/generator/vanilla/VanillaCompatibilityGenerator.java

```diff
@@ -261,6 +269,8 @@ public CubePrimer generateCube(int cubeX, int cubeY, int cubeZ) {
     @Override
     public void populate(ICube cube) {
         tryInit(vanilla, world);
+        Random rand = getChunkSpecificRandom(cube.getX(), cube.getZ());
```

Comment:
Why not make the seed change with cube Y here too? It doesn't make sense to do for actual vanilla generator, but it does make sense for CC.